app/workers/metrics/record_daily_usage_measurables_worker.rb

```diff
@@ -0,0 +1,17 @@
+module Metrics
+  class RecordDailyUsageMeasurablesWorker
```

Comment:
I would remove `Measurables`, it's already in the `Metrics` namespace

Reply:
Done